### ISSUE TITLE
message() should not return a trailing CR

### ISSUE THREAD

**Issue by FGasper** [libwww-perl/HTTP-Message#87]
```
> perl -MHTTP::Response -MData::Dumper -e'$Data::Dumper::Useqq = 1; print Dumper( HTTP::Response->parse("HTTP/1.0 200 OK\x0d\x0aContent-type: text/plain\x0d\x0a\x0d\x0aHi.")->message() )'
$VAR1 = "OK\r";
```

**Comment by FGasper** [comment]
Regression test added.

**Comment by oalders** [comment]
Closed via #88
